eras/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/Rules/Utxo.hs

```diff
@@ -337,13 +337,18 @@ utxoTransition = do
   let refunded = Shelley.keyRefunds pp txb
   let txCerts = toList $ getField @"certs" txb
   let depositChange = totalDeposits pp (`Map.notMember` stakepools) txCerts Val.<-> refunded
+  let utxoAdd = txouts txb -- These will be inserted into the UTxO
+  let utxoDel = eval (txins @era txb ◁ utxo) -- These will be deleted from the UTxO
+  let newIncStakeDistro = updateStakeDistribution @era incStake utxoDel utxoAdd
 
   pure
     Shelley.UTxOState
-      { Shelley._utxo = eval ((txins @era txb ⋪ utxo) ∪ txouts txb),
+      { Shelley._utxo = eval ((txins @era txb ⋪ utxo) ∪ utxoAdd),
+        -- Domain exclusion (a ⋪ b) deletes 'a' from the domain of 'b'
```

Comment:
```suggestion
```

This is a standard function used in various places, and the documentation can be seen through haddock etc

Reply:
done